oidc-controller/signing_key.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
+MIIEvQIBADANBgkqhkiG9w0BAQEFAASCBKcwggSjAgEAAoIBAQDWlFWWa1WU7gHA
+lbXZzdEGYGsZFq16WYPG/vglGGVHfpoxInDuzUvFv5etx9dGYpEWsjKfQUKSa0oZ
+RO/v/M3ksqGrk9of8MUxSCZ7zUYy96leFd1sV/QCDpPFv3Pse9e0cLsf0AgidMHo
+TTVbFlrRixVD4NVU9FxBtQwA0u5fyetSMPFxS8gdcuX5n/DZUyBNk37UfW5VXw5g
+Mz/Ql5yTyE8QitO3CFjO/E30Q/x5kjsFAwjETkHozL0Q8rNhpPu8oDeBIYYndY6z
+gBODZYDblif9Ngv0lPpvsX6JMTvl8euJQyh/syJSvtxHz3Gf0FLiJvdCyN2qg7+O
+YiyUWJ3hAgMBAAECggEAGu139R2omn0otqKC9N91buusHPSh1gQEO7V07fDLCucK
+EgbsracRLnJt5ia5LvHBG9xGaoJNyKKSpKuoQZFTxDKfg7DU82OrV1Ui0S/wMoau
+WCp9xDIgYp6yMCNnuIJNDi2EaghY8Pv+dniz7KeYQ4oE4+UD2Hqm1i8SX8sYk2gA
+UPvjGUIrktaWPyQ+OnFy4ET9q4UTE8wWyzaIrV2Jtw7gq2OZZW7AI9zBmaXYoMd7
+7hz9od10xkSikg2CxHtVHnNbv7vi6rs1GOwa3rS5N6AZy8smb/qzTq+rEUnjEC2w
+WoEy4h5Q3TJOiQEpaHrdbCF1fwDXVfQHfxQlPVvxzQKBgQDgeRBR02bbVO7izsw3
+99ujtS308WpyZPhJt/3lTyiKAp8PANYN/y8QIMbZ8VUgFAcIkNY9AqcXFDvnxeIb
+HhALUDosN3AsFgL/IBb0K2l63f7PpU2CvaWIdtoYOTuQeJZRopYGSB+ov+jOJgPU
+0a2aR6YjznXkEVg1u/DerBtivQKBgQD0t4zV5gYUtjUt7+UyT5kUL9/xAw7LYlyW
+MJ9AqEoxF6D9kJa7ta5LbIp2ow3q4+7sRh/W2aBMorPPVZFb7nHw46oHLRkWUMm7
+GINcj80F+NGcQFC+WFqpn6l4keeS0aKGKwPXuhxTytBGGuT97R9O0ySxSw3eiBfU
+FrtYvUkL9QKBgG8Z3IbzYCRMvFPL0O4eojo7sXxbhLkL2ov6IhM+A62TMJCndErH
+M88Cugki1mQjJXwK286konjcze2gVpHfZkeDNNevtLhNSLxncJT3a6Shfb3ebie4
+n9aUJMv6f824sYXEOAbx6aGHIr3tuNW71fXXYpa0WEtCfmtSExzTCFbtAoGAJY6N
+9L3qrP8Rxk0M9d21dp2qVsjLWndsEjjAGCI70LD5qjCokYJej4DG91Iq7qOwpV5V
+wJmBuoE161oMhvlfT0MBczx+nygi7a3jay8+ne4D7WEJg0z4DRRN5A0wZx35PXIY
+kuEjO9Diif/SqQ3dQjGxbexA4qgsUAZ1FjdmoIkCgYEAk/NALy0yZvGVesD6A8a9
+ia8XzKSs3atjhZNDizkvvBih0R7n+56uzeTrows0WSMlv/15UmiY6iFfeLmw/jDh
+8qL+v34OPbDbAePP1CEdtaoKKjj6YVeckcOmUb0rYErPv4PbHIIiERdsmZMnaorW
+jfl+/i321T59vRlAreiPXLU=
+-----END PRIVATE KEY-----
```

Comment:
This file should be .gitignored: teh framework should generate it on first start, if not available.

Reply:
unversioned